weave-js/src/util/llmTokenCosts.ts

```diff
@@ -3,106 +3,109 @@ export const isValidLLMModel = (model: string): model is Model => {
   return model in Object.keys(LLM_TOKEN_COSTS);
 };
 
-export const LLM_TOKEN_COSTS: Record<string, {input: number; output: number}> =
```

Comment:
getting rid of the this typing allows the above type, `keyof typeof LLM_TOKEN_COSTS` to correctly be set, instead of just being string

Reply:
I think something like this should also work, but I'm ok with it as is.
```
type KnownModel =
  | 'gpt-4o'
  | 'gpt-4o-2024-05-13'
  | 'gpt-4o-2024-05-13'
  | 'gpt-4-turbo-2024-04-09'
  ...

type Costs = {
  input: number;
  output: number;
};

export const LLM_TOKEN_COSTS: Record<KnownModel | 'default', Costs> = {
```